preinst

```diff
@@ -31,10 +31,12 @@ if [ "$PACKAGEUPGRADE" = "true" ] ; then
     touch $UPGRADE_TEMP_FILE
 fi
 
+# TODO What should happen with these?
 if [ -f "/var/lib/irods/VERSION" ] ; then
     mv /var/lib/irods/VERSION /var/lib/irods/VERSION.previous
 fi
 
+# TODO What should happen with these?
 if [ -f "/var/lib/irods/VERSION.json" ] ; then
     mv /var/lib/irods/VERSION.json /var/lib/irods/VERSION.json.previous
 fi
```

Comment:
Maybe all we do here is make the second argument lowercase.

That seems to be what we would want given that this is the pre-install script. Then everything would fall in line for the startup script since it would be looking for the lowercase filename.